<?xml version="1.0"?>

<!-- This file was auto-generated by the ros package drone_middleware -->

<launch>
	<arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor.gazebo.xacro" />
	
	<group ns="drone0">
		<include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
			<arg name="name" value="drone0" />
			<arg name="tf_prefix" value="drone0" />
			<arg name="model" value="$(arg model)" />
			<arg name="x" value="0.0" />
			<arg name="y" value="0.0" />
		</include>
	</group>

</launch>

